### PR TITLE
Simple crud project

### PR DESCRIPTION
**Pull request by AnvarjonShavqiyev**

### CODE REVIEW

File: my-nest-project/package.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    "@nestjs/core": "^10.0.0",
+    "@nestjs/platform-express": "^10.0.0",
+    "@nestjs/typeorm": "^10.0.2",
+    "pg": "^8.12.0",
+    "reflect-metadata": "^0.2.0",
+    "rxjs": "^7.8.1",
+    "typeorm": "^0.3.20"
+  },
+  "devDependencies": {
+    "@nestjs/cli": "^10.0.0",
```
Comment (esnts) [inline my-nest-project/package.json:33]: We are not using version auto-incrementing `^`

File: my-nest-project/src/app.module.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { User } from './entites/user.entity';
+import { UserController } from './controllers/user.controller';
+
+@Module({
+  imports: [
+    TypeOrmModule.forRoot({
+      type: 'postgres',
+      url: "postgres://default:ZnU1ztG6ocdg@ep-white-snow-a47lu56s-pooler.us-east-1.aws.neon.tech:5432/verceldb?sslmode=require",
```
Comment (esnts) [inline my-nest-project/src/app.module.ts:10]: NEVER push credentionals.

File: my-nest-project/src/controllers/user.controller.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        return user;
+    }
+
+    @Post()
+    async createUser(@Body() user: User): Promise<User> {
+        try {
+            const newUser = await this.userRepository.save(user);
+            return newUser;
+        } catch (error) {
+            throw new BadRequestException('Failed to create user');
```
Comment (esnts) [inline my-nest-project/src/controllers/user.controller.ts:35]: move all common errors/strings to file with constants

File: my-nest-project/package.json (hunk 1 above)
Comment (AnvarjonShavqiyev) [reply my-nest-project/package.json:33]: i got it

File: my-nest-project/src/app.module.ts (hunk 2 above)
Comment (AnvarjonShavqiyev) [reply my-nest-project/src/app.module.ts:10]: I know about it, in these time i found vercel to free deployment, it was very interesting and i forget improve it.
I never push credentionls, i am sorry

File: my-nest-project/.env [hunk 4]
```diff
@@ -0,0 +1 @@
+DATABASE_URL="postgres://default:ZnU1ztG6ocdg@ep-white-snow-a47lu56s-pooler.us-east-1.aws.neon.tech:5432/verceldb?sslmode=require"
```
Comment (EugeneSinitsa) [inline my-nest-project/.env:1]: Create `.env.template` with  
```DATABASE_URL="..."```

**Review by EugeneSinitsa — CHANGES_REQUESTED** [review]

**Comment by esnts** [comment]
I still can find all credentials into your commits history 
Screenshot 2024-07-04 at 12 07 42


You can merge it after commits history clean up

**Pull request closed without merging**